imsv-docs-docusaurus/openapi/endpoints/login/models/siwe-login-request.yaml

```diff
@@ -20,6 +20,7 @@ properties:
     enum:
       - polygon-mainnet
       - polygon-amoy
+      - eth-sepolia
```

Comment:
instead of putting the enum here, i would reccomend pointing them to the supported chains page. This enum used on algorand and xrpl was an interim solution since those chains did not have a protocol defined on the docs yet

Reply:
@tonykha123 what do you think about doing both (keep the enum and also link to supported chains)? At least until the list of supported evm chains becomes too long to display inline, and then we could remove the enum and leave the link to supported chains?

Reply:
I'm fine with either option though, removing the enum or keeping it, I just think it's more consistent to keep it while we have it for Algo and XRPL?

Reply:
sure we can keep both. will have to maintain both, itll be easy to forget though